Sources/Rendering/Core/ColorTransferFunction/index.d.ts

```diff
@@ -257,6 +267,11 @@ export interface vtkColorTransferFunction extends vtkObject {
    */
   setMappingRange(min: number, max: number): void;
 
+  /**
+   * Get the range of mapped scalars.
+   */
+  getRange(): number[];
```

Comment:
should be defined in superclass vtkScalarsToColors

Reply:
It should be fixed in vtkScalarsToColors, the `min` and `max` parameters are ignored by the function. 

I think they should be removed from `ScalarsToColors/index.d.ts` but also from `ScalarsToColors/index.js` (in a different commit, same PR)